### ISSUE TITLE
[BUG] Unable to set alpha of add_brain_region when running from Jupyter Notebook 

### ISSUE THREAD

**Issue by harshk95** [brainglobe/brainrender#390]
I am unable to set the alpha of the brain region added via the alpha argument. Without the argument set one can see with alpha = 1, however, for any other value of alpha, it appears as if alpha were set at 0. 

This is for _**Brainrender v2.1.9**_

For example, running the example notebook workflow, 
```

import vedo
vedo.settings.default_backend= 'vtk'

from brainrender import Scene
popup_scene = Scene(atlas_name='allen_mouse_50um', title='popup')

popup_scene.add_brain_region('VISp', alpha = 0.99)

popup_scene.render() 
```

yields 
![image](https://github.com/user-attachments/assets/a2241173-1092-40c6-b536-8618b1762c3b)

while running
```
import vedo
vedo.settings.default_backend= 'vtk'

from brainrender import Scene
popup_scene = Scene(atlas_name='allen_mouse_50um', title='popup')

popup_scene.add_brain_region('VISp', alpha = 1)

popup_scene.render() 

```

yields 

![image](https://github.com/user-attachments/assets/3c1ceb31-9006-4145-97b1-e63e8bd18a8b)

- OS: Windows Server 2022
- Hardware specs Intel Xeon Platinum 8260, 384GB RAM

**Comment by alessandrofelder** [comment]
Hi @harshk95 
Thanks for reporting - I ran 2.1.9 on my Macbook and couldn't reproduce this issue, so it might be a Windows thing - or a bug in the dependencies... not sure.

Could you try updating your `brainrender` and your `vedo` by running
```bash
pip install -U brainrender
pip install -U vedo
```
And let us know if you still encounter the problem, please?

**Comment by harshk95** [comment]
Hi thanks for the reply, 

despite running the updates, the issue seems to persist. 

Brainrender v 2.1.9
vedo v 2024.5.2

**Comment by adamltyson** [comment]
Hi @harshk95, it looks like brainrender hasn't been updated. Was your conda environment activated when you ran the installation command?

**Comment by harshk95** [comment]
Hi @adamltyson , it was indeed in the environment. However, I had python 3.9 which was only supported by brainrender 2.1.9. I recreated the environment with python 3.10 which is required by the latest brainrender distribution, but the issue persists. Pasted below is the pip list of the environment for confirmation.

> Package                       Version

> ----------------------------- --------------
> absl-py                       2.1.0
> aiohappyeyeballs              2.4.3
> aiohttp                       3.10.10
> aiosignal                     1.3.1
> alabaster                     1.0.0
> allensdk                      2.16.2
> anndata                       0.10.9
> annotated-types               0.7.0
> anyio                         4.6.2.post1
> app-model                     0.3.0
> appdirs                       1.4.4
> apptools                      5.3.0
> argon2-cffi                   23.1.0
> argon2-cffi-bindings          21.2.0
> argschema                     3.0.4
> array_api_compat              1.9
> arrow                         1.3.0
> asciitree                     0.3.3
> asttokens                     2.4.1
> async-lru                     2.0.4
> async-timeout                 4.0.3
> attrs                         24.2.0
> babel                         2.16.0
> bctpy                         0.6.1
> beautifulsoup4                4.12.3
> bleach                        6.1.0
> blosc2                        2.7.1
> boto3                         1.35.49
> botocore                      1.35.49
> brainglobe                    1.3.1
> brainglobe-atlasapi           2.0.9
> brainglobe-heatmap            0.5.4
> brainglobe-napari-io          0.3.4
> brainglobe-segmentation       1.2.6
> brainglobe_space              1.0.2
> brainglobe-utils              0.6.2
> brainreg                      1.0.10
> brainrender                   2.1.15
> brainrender-napari            0.0.4.post1
> bs4                           0.0.2
> build                         1.2.2.post1
> cachetools                    5.5.0
> cachey                        0.2.1
> ccf_streamlines               1.1.4
> cebra                         0.4.0
> cellfinder                    1.3.3
> certifi                       2024.8.30
> cffi                          1.17.1
> charset-normalizer            3.4.0
> click                         8.1.7
> cloudpickle                   3.1.0
> cmcrameri                     1.9
> colorama                      0.4.6
> comm                          0.2.2
> configobj                     5.0.9
> contourpy                     1.3.0
> cupy                          12.1.0
> cycler                        0.12.1
> dask                          2024.8.1
> debugpy                       1.8.7
> decorator                     4.4.2
> defusedxml                    0.7.1
> distro                        1.9.0
> docstring_parser              0.16
> docutils                      0.21.2
> envisage                      7.0.3
> exceptiongroup                1.2.2
> executing                     2.1.0
> fancylog                      0.4.3
> fasteners                     0.19
> fastjsonschema                2.20.0
> fastrlock                     0.8.2
> ffmpeg-python                 0.2.0
> filelock                      3.16.1
> flexcache                     0.3
> flexparser                    0.3.1
> fonttools                     4.54.1
> fqdn                          1.5.1
> freetype-py                   2.5.1
> frozenlist                    1.5.0
> fsspec                        2024.10.0
> future                        1.0.0
> Glymur                        0.13.6
> google                        3.0.0
> greenlet                      3.1.1
> h11                           0.14.0
> h5py                          3.12.1
> hdmf                          3.14.5
> HeapDict                      1.0.1
> hsluv                         5.0.4
> httpcore                      1.0.6
> httpx                         0.27.2
> idna                          3.10
> imagecodecs                   2024.9.22
> imageio                       2.36.0
> imageio-ffmpeg                0.5.1
> imagesize                     1.4.1
> imbalanced-learn              0.12.4
> importlib_metadata            8.5.0
> importlib_resources           6.4.5
> in-n-out                      0.2.1
> ipykernel                     6.29.5
> ipython                       8.29.0
> ipywidgets                    8.1.5
> isoduration                   20.11.0
> jax                           0.4.30
> jaxlib                        0.4.30
> jaxopt                        0.8.3
> jedi                          0.19.1
> Jinja2                        3.1.4
> jmespath                      1.0.1
> joblib                        1.4.2
> json5                         0.9.25
> jsonpointer                   3.0.0
> jsonschema                    4.23.0
> jsonschema-specifications     2024.10.1
> jupyter_client                8.6.3
> jupyter_core                  5.7.2
> jupyter-events                0.10.0
> jupyter-lsp                   2.2.5
> jupyter_server                2.14.2
> jupyter_server_terminals      0.5.3
> jupyterlab                    4.2.5
> jupyterlab_pygments           0.3.0
> jupyterlab_server             2.27.3
> jupyterlab_widgets            3.0.13
> k3d                           2.16.1
> keras                         3.5.0
> kiwisolver                    1.4.7
> lazy_loader                   0.4
> legacy-api-wrap               1.4
> literate-dataclasses          0.0.6
> llvmlite                      0.43.0
> locket                        1.0.0
> loguru                        0.7.2
> lxml                          5.3.0
> magicgui                      0.9.1
> markdown-it-py                3.0.0
> MarkupSafe                    3.0.2
> marshmallow                   3.23.0
> matplotlib                    3.9.2
> matplotlib-inline             0.1.7
> matplotlib-venn               1.1.1
> mayavi                        4.8.2
> mdurl                         0.1.2
> meshio                        5.3.5
> mistune                       3.0.2
> ml_dtypes                     0.5.0
> morphapi                      0.3.0
> morphio                       3.4.0
> moviepy                       1.0.3
> mpmath                        1.3.0
> msgpack                       1.1.0
> multidict                     6.1.0
> myterial                      1.2.1
> namex                         0.0.8
> napari                        0.5.4
> napari-console                0.1.0
> napari-ndtiffs                0.2.1
> napari-plugin-engine          0.2.0
> napari-plugin-manager         0.1.3
> napari-svg                    0.2.0
> natsort                       8.4.0
> nbclient                      0.10.0
> nbconvert                     7.16.4
> nbformat                      5.10.4
> ndindex                       1.9.2
> ndx-events                    0.2.0
> nemos                         0.1.6
> nest-asyncio                  1.6.0
> networkx                      3.4.2
> neurom                        3.2.11
> nibabel                       5.3.2
> notebook_shim                 0.2.4
> npe2                          0.7.7
> nptyping                      2.5.0
> npyx                          4.0.6
> numba                         0.60.0
> numcodecs                     0.13.1
> numexpr                       2.10.1
> numpy                         1.23.5
> numpydoc                      1.8.0
> opencv-python                 4.10.0.84
> opt_einsum                    3.4.0
> optree                        0.13.0
> overrides                     7.7.0
> packaging                     24.1
> pandas                        1.5.3
> pandocfilters                 1.5.1
> parso                         0.8.4
> partd                         1.4.2
> patsy                         0.5.6
> pillow                        11.0.0
> Pint                          0.24.3
> pip                           24.2
> platformdirs                  4.3.6
> pooch                         1.8.2
> proglog                       0.1.10
> prometheus_client             0.21.0
> prompt_toolkit                3.0.48
> propcache                     0.2.0
> psutil                        6.1.0
> psycopg2-binary               2.9.10
> psygnal                       0.11.1
> pure_eval                     0.2.3
> py                            1.11.0
> py-cpuinfo                    9.0.0
> pyarrow                       18.0.0
> pyconify                      0.1.6
> pycparser                     2.22
> pydantic                      2.9.2
> pydantic-compat               0.1.2
> pydantic_core                 2.23.4
> pyface                        8.0.0
> Pygments                      2.18.0
> pyinspect                     0.1.0
> pynapple                      0.7.1
> pynndescent                   0.5.13
> pynrrd                        1.0.0
> pynwb                         2.8.2
> PyOpenGL                      3.1.7
> pyparsing                     3.2.0
> pyproject_hooks               1.2.0
> PyQt5                         5.15.11
> PyQt5-Qt5                     5.15.2
> PyQt5_sip                     12.15.0
> python-dateutil               2.9.0.post0
> python-json-logger            2.0.7
> pytz                          2024.2
> pywin32                       308
> pywinpty                      2.0.14
> PyYAML                        6.0.2
> pyzmq                         26.2.0
> qtconsole                     5.6.0
> QtPy                          2.4.1
> referencing                   0.35.1
> requests                      2.32.3
> requests-toolbelt             1.0.0
> retry                         0.9.2
> rfc3339-validator             0.1.4
> rfc3986-validator             0.1.1
> rich                          13.9.3
> rpds-py                       0.20.0
> ruamel.yaml                   0.18.6
> ruamel.yaml.clib              0.2.12
> s3transfer                    0.10.3
> scanpy                        1.10.3
> scikit-build                  0.18.1
> scikit-image                  0.24.0
> scikit-learn                  1.5.2
> scipy                         1.10.0
> seaborn                       0.13.2
> semver                        3.0.2
> Send2Trash                    1.8.3
> session_info                  1.0.0
> setuptools                    57.0.0
> shellingham                   1.5.4
> SimpleITK                     2.4.0
> simplejson                    3.19.3
> six                           1.16.0
> slurmio                       0.1.1
> sniffio                       1.3.1
> snowballstemmer               2.2.0
> soupsieve                     2.6
> Sphinx                        8.1.3
> sphinxcontrib-applehelp       2.0.0
> sphinxcontrib-devhelp         2.0.0
> sphinxcontrib-htmlhelp        2.1.0
> sphinxcontrib-jsmath          1.0.1
> sphinxcontrib-qthelp          2.0.0
> sphinxcontrib-serializinghtml 2.0.0
> SQLAlchemy                    2.0.36
> stack-data                    0.6.3
> statsmodels                   0.14.4
> stdlib-list                   0.11.0
> superqt                       0.6.7
> sympy                         1.13.1
> tables                        3.10.1
> tabulate                      0.9.0
> tdt                           0.6.6
> terminado                     0.18.1
> threadpoolctl                 3.5.0
> tifffile                      2024.9.20
> tinycss2                      1.4.0
> tomli                         2.0.2
> tomli_w                       1.1.0
> toolz                         1.0.0
> torch                         2.5.0
> tornado                       6.4.1
> tqdm                          4.66.5
> traitlets                     5.14.3
> traits                        6.4.3
> traitsui                      8.0.0
> traittypes                    0.2.1
> treelib                       1.7.0
> triangle                      20230923
> typer                         0.12.5
> types-python-dateutil         2.9.0.20241003
> typing_extensions             4.12.2
> umap-learn                    0.5.6
> uri-template                  1.3.0
> urllib3                       2.2.3
> vedo                          2024.5.2
> vispy                         0.14.3
> vtk                           9.3.1
> wcwidth                       0.2.13
> webcolors                     24.8.0
> webencodings                  0.5.1
> websocket-client              1.8.0
> wheel                         0.44.0
> widgetsnbextension            4.0.13
> win32-setctime                1.1.0
> wrapt                         1.16.0
> xarray                        2023.1.0
> yarl                          1.16.0
> zarr                          2.18.2
> zipp                          3.20.2

**Comment by alessandrofelder** [comment]
Hey @harshk95 
Can you run the commands below in the same notebook, and post the output, please:
```python
print(vedo.settings)

from brainrender import settings
print(settings.ROOT_ALPHA)
```

**Comment by alessandrofelder** [comment]
Sure, if you report any observed patterns that could help, so you're more than welcome to!
